unit test //tensorflow/compiler/xla/service/cpu/tests:cpu_literal_caching_test fails to build on AARCH64

<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 20.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: n/a
- TensorFlow installed from (source or binary): source
- TensorFlow version: git HEAD
- Python version: 3.6.9
- Installed using virtualenv? pip? conda?: no
- Bazel version (if compiling from source): 3.7.2
- GCC/Compiler version (if compiling from source): 10.3.0
- CUDA/cuDNN version: n/a
- GPU model and memory: n/a



**Describe the problem**

Build fails with
Execution platform: @local_execution_config_platform//:platform
In file included from ./tensorflow/compiler/xla/service/backend.h:31,
                 from ./tensorflow/compiler/xla/tests/hlo_test_base.h:26,
                 from ./tensorflow/compiler/xla/tests/codegen_test_base.h:24,
                 from ./tensorflow/compiler/xla/tests/llvm_irgen_test_base.h:22,
                 from ./tensorflow/compiler/xla/service/cpu/tests/cpu_codegen_test.h:19,
                 from tensorflow/compiler/xla/service/cpu/tests/cpu_literal_caching_test.cc:18:
./tensorflow/compiler/xla/service/transfer_manager.h:37:1: error: expected unqualified-id before 'namespace'
   37 | namespace xla {
      | ^~~~~~~~~
In file included from ./tensorflow/compiler/xla/tests/hlo_test_base.h:26,
                 from ./tensorflow/compiler/xla/tests/codegen_test_base.h:24,
                 from ./tensorflow/compiler/xla/tests/llvm_irgen_test_base.h:22,
                 from ./tensorflow/compiler/xla/service/cpu/tests/cpu_codegen_test.h:19,
                 from tensorflow/compiler/xla/service/cpu/tests/cpu_literal_caching_test.cc:18:
./tensorflow/compiler/xla/service/backend.h:97:3: error: 'TransferManager' does not name a type
   97 |   TransferManager* transfer_manager() const { return transfer_manager_; }
      |   ^~~~~~~~~~~~~~~
./tensorflow/compiler/xla/service/backend.h:164:11: error: 'TransferManager' has not been declared
  164 |           TransferManager* transfer_manager,
      |           ^~~~~~~~~~~~~~~
./tensorflow/compiler/xla/service/backend.h:172:3: error: 'TransferManager' does not name a type
  172 |   TransferManager* transfer_manager_;
      |   ^~~~~~~~~~~~~~~
INFO: Elapsed time: 707.544s, Critical Path: 172.52s
INFO: 4160 processes: 63 internal, 4097 local.
FAILED: Build did NOT complete successfully


**Provide the exact sequence of commands / steps that you executed before running into the problem**

$ bazel test --test_timeout=300,500,-1,-1 --flaky_test_attempts=3 --test_output=all --cache_test_results=no --remote_http_cache=""  --remote_cache_proxy="" --noremote_accept_cached --config=nonccl --build_tag_filters=-no_oss,-oss_serial,-gpu,-tpu,-benchmark-test,-v1only --test_tag_filters=-no_oss,-oss_serial,-gpu,-tpu,-benchmark-test,-v1only --copt=-ffp-contract=off --cxxopt=-ffp-contract=off --copt=-Og --copt=-ggdb --cxxopt=-Og --cxxopt=-ggdb --verbose_failures -- //tensorflow/compiler/xla/service/cpu/...

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

--- a/tensorflow/compiler/xla/service/cpu/test_target_triple_helper.h
+++ b/tensorflow/compiler/xla/service/cpu/test_target_triple_helper.h
@@ -18,7 +18,7 @@ limitations under the License.
 
 #if defined(__aarch64__)
 static const char kTargetCpuForHost[] = "aarch64";
-static const char kTargetTripleForHost[] = "aarch64-unknown-linux-gnu",
+static const char kTargetTripleForHost[] = "aarch64-unknown-linux-gnu";
 #elif (defined(__powerpc__) || \
        defined(__ppc__) && (__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__))
 static const char kTargetCpuForHost[] = "ppc";

